faqs.rst

```diff
@@ -20,3 +20,82 @@ How do I report a bug?
 ----------------------
 
 You can read up on reporting bugs here: :ref:`contributing` or report one directly for flux `core <https://github.com/flux-framework/flux-core/issues>`_ or `sched <https://github.com/flux-framework/flux-sched/issues>`_.
+
+.. _not_managing_all_resources:
+
+---------------------------------------------------------------------------------
+Why is Flux not discovering and managing all of the resources on the system/node?
+---------------------------------------------------------------------------------
+
+* Various bind flags passes to the parallel launcher that started Flux
+* CUDA enabled hwloc (for GPUs)
+
+.. _launch_large_num_jobs:
+
+------------------------------------------------------------
+How do I efficiently launch a large number of jobs to Flux?
+------------------------------------------------------------
+
+See `bulksumbission <https://github.com/flux-framework/flux-workflow-examples/tree/master/async-bulk-job-submit>`_ for an example workflow or flux `tree <https://github.com/flux-framework/flux-sched/blob/master/src/cmd/flux-tree>`_.
+
+.. _node_memory_exhaustion:
+
+------------------------------------------------------------------
+Memory exhaustion on a node when running large ensembles with Flux
+------------------------------------------------------------------
+
+Set ``rundir`` so that the content store is not saved to ``/tmp`` but to filesystem.
+
+.. _byop:
+
+-----------------------------
+Bring Your Own Python (BYOP)
+-----------------------------
+
+Caveats on what will work and what won't swork when using a version of python that isn't the one Flux was configured against.
+
+.. _mimic_slurm_jobstep:
+
+----------------------------------------
+How do I mimic Slurm's jobstep semantics
+----------------------------------------
+
+``flux mini submit`` ting a script with ``flux mini run`` s will not result in Slurm-styel jobstes unless you stick ``flux-start`` in from the jobscript in ``flux mini sumbit`` .
+
+.. _mpi_bootstrap_fails:
+
+---------------------------------------------------------------------------------------------
+Flux is failing to bootstrap a specific MPI implementaion (e.g. OpenMPI, MPICH, Spectrum MPI)
+---------------------------------------------------------------------------------------------
+
+Use the ``-o mpi=`` option when running or submitting jobs.
+
+.. _message_callback_not_run:
+
+-----------------------------------------------------
+My message callback is not being run. How do I debug?
+-----------------------------------------------------
+
+* Check the error codes from ``addvec`` , ``register_service`` , the ``RPC`` , etc
```

Comment:
I'd add a `TODO` comment that we should link these to the man pages once the flux-core man pages are included in readthedocs.

Reply:
Unfortunately I was being lazy when jotting these notes down, so they aren't full function names either.  Can we expand them?  For example `RPC` -> `flux_rpc*` and `register_service` -> `flux_register_service`.